debian/changelog

```diff
@@ -1,3 +1,37 @@
+revpi-tools (2.0.1-1+revpi11+1) bullseye; urgency=medium
```

Comment:
How is this a 3.0.0-1 release?

Reply:
I wanted to do this fast before our planning... :-1: I will avoid this again.

The tag is now in my Repo -> [tag v3.0.0](https://github.com/RamiGspo/revpi-tools/releases/tag/v3.0.0)

Reply:
Sorry @RamiGspo, I told you, let's start the version 3.0.0 at that point. For version 3.0.0, two other PullRequests are missing, which must first be merged before the main branch is 3.0.0. Let's wait for it and then drive up with this PR.